web-portal/components/EventView.vue

```diff
@@ -212,6 +223,9 @@
         console.error('Action:', e.action)
         console.error('Trigger:', e.trigger)
         window.alert('We were unable to copy URL. Here it is for reference:\n' + this.event.bitly_link)
+      },
+      adjustDatetimeForTimezone(datetime, timezone) {
+        return momenttz(datetime).tz(timezone)
```

Comment:
I don't see any other references to this in the PR. If it ended up not being used let's remove it.

Reply:
good catch! done, thanks ✅